python/lsst/obs/subaru/instrument.py

```diff
@@ -49,13 +48,15 @@ class HyperSuprimeCam(Instrument):
     """Gen3 Butler specialization class for Subaru's Hyper Suprime-Cam.
     """
 
+    policyName = "hsc"
+    obsDataPackage = "obs_subaru_data"
     filterDefinitions = HSC_FILTER_DEFINITIONS
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
         packageDir = getPackageDir("obs_subaru")
         self.configPaths = [os.path.join(packageDir, "config"),
-                            os.path.join(packageDir, "config", "hsc")]
+                            os.path.join(packageDir, "config", self.policyName)]
```

Comment:
Per my comment in the obs_base PR, `self.policyName` here could probably be replaced with `self.name.lower()`.